src/services/codefixes/parameterPropertyToDeclaration.ts

```diff
@@ -0,0 +1,90 @@
+/* @internal */
+namespace ts.codefix {
+    interface ParameterPropertyToPropertyInfo {
+        parameter: ParameterPropertyDeclaration
+        ctor: ConstructorDeclaration
+        container: ClassLikeDeclaration

```

Comment:
Will `container` always be the parent of `ctor`.  If so, is there value in storing it separately?